### PR TITLE
Use overtyped headline if available

### LINKED ISSUE #2118: Use overtyped headline if available
**Is your feature request related to a problem? Please describe.**
CPS provides a 'overtyped' key, which were meant to use if availible. Why do they even give you the normal headline? Good question. Because they like to see us suffer :)

```
"headlines": {
   "headline": "Presidential Election Tribunal reject PDP request to inspect INEC server",
   "overtyped": "Presidential Tribunal reject Atiku/PDP request"
},
```

**Describe the solution you'd like**
Use the `overtyped` key if available, else use `headline`

**Testing notes**
[Tester to complete]

Dev insight: Unit tests

**Additional context**

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves #2118 

**Overall change:** _Use overtyped headline in storypromo if `overtyped` is provided else use the provided headline_

**Code changes:**

- _Render `overtyped` if `overtyped` is defined else render the `headline` provided ._=
---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/containers/StoryPromo/LinkContents.jsx [hunk 1]
```diff
@@ -15,10 +15,10 @@ const LinkContents = ({ item }) => {
   } = useContext(ServiceContext);
 
   const isMedia = deepGet(['cpsType'], item) === 'MAP';
-  const headline = deepGet(['headlines', 'headline'], item);
-
+  const headlines = deepGet(['headlines'], item);
+  const { headline, overtyped } = headlines;
   if (!isMedia) {
-    return headline;
+    return overtyped || headline;
```
Comment (ghost) [inline src/app/containers/StoryPromo/LinkContents.jsx]: Could we extract this to a variable like `contents` and use it on line `38` and `21`

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]
Looks good! 👍

**Review by pjlee11 — APPROVED** [review]
LGTM

**Comment by david-boydell** [comment]
Looks good, this is ready for merge.

**Pull request merged**